build.gradle.kts

```diff
@@ -51,6 +51,8 @@ kotlin {
         implementation("io.kotest:kotest-property:5.6.1")
         implementation("io.kotest:kotest-framework-engine:5.6.1")
         implementation("io.kotest:kotest-assertions-core:5.6.1")
+        implementation("io.kotest:kotest-framework-datatest:5.6.1")
+        implementation("io.kotest:kotest-runner-junit5:5.6.1")
```

Comment:
`kotest-runner-junit` must be defined in `jvmTest` section

Reply:
Thank you @ustits! I'm bad with Gradle will move it and hope that it'll fix it 👍